src/main/java/org/broadinstitute/hellbender/tools/reference/CompareReferences.java

```diff
@@ -204,6 +256,193 @@ private void displayBySequenceName(Set<ReferenceSequenceTable.TableRow> rows, St
         System.out.println();
     }
 
+    private void runFullAlignment(ReferencePair refPair, ReferenceSequenceTable table){
+        List<File> snpsFiles = new ArrayList<>();
+        if(refPair.getAnalysis().contains(ReferencePair.Status.DIFFER_IN_SEQUENCE)) {
+            // find the mismatch sequence
+            for (String sequenceName : table.getAllSequenceNames()) {
+                Set<ReferenceSequenceTable.TableRow> rows = table.queryBySequenceName(sequenceName);
+                if (rows.size() == 2) {
+                    // generate fasta files for mismatching sequences
+                    GATKPath ref1Path = refPair.getRef1();
+                    GATKPath ref2Path = refPair.getRef2();
+
+                    String sequenceInRef1Name = ref1Path.toPath().getFileName() + "." + sequenceName;
+                    String sequenceInRef2Name = ref2Path.toPath().getFileName() + "." + sequenceName;
+
+                    File ref1TempFastaOutput = IOUtils.createTempFile(sequenceInRef1Name, ".fasta");
+                    File ref2TempFastaOutput = IOUtils.createTempFile(sequenceInRef2Name, ".fasta");
+
+                    GATKPath ref1Fasta = generateFastaForSequence(ReferenceDataSource.of(ref1Path.toPath(), true), sequenceName, new GATKPath(ref1TempFastaOutput.toString()));
+                    GATKPath ref2Fasta = generateFastaForSequence(ReferenceDataSource.of(ref2Path.toPath(), true), sequenceName, new GATKPath(ref2TempFastaOutput.toString()));
+
+                    // pass fastas into mummer, get back a vcf and add to list
+                    MummerExecutor executor = new MummerExecutor();
+                    logger.info("Running mummer alignment on sequence " + sequenceName);
+                    File tempSnpsDirectory = IOUtils.createTempDir("tempsnps");
```

Comment:
so you are making a whole new tmpdir within each of these loops for every mismatching contig... This can get to be a lot potentially and these are all java tmpdirs that close on the jvm exit too (with apprxoimately 2 uncompressed references' worth of inputs in here....
I think this method should probably manage the deletion of the tmp dir itself to cut down on disk usage. @droazen thoughs?